rllib/offline/offline_data.py

```diff
@@ -70,6 +70,8 @@ def __init__(self, config: AlgorithmConfig):
             logger.error(e)
         # Avoids reinstantiating the batch iterator each time we sample.
         self.batch_iterator = None
+        # Defines the prelearner class. Note, this could be user-defined.
+        self.prelearner_class = self.config.get("prelearner_class", OfflinePreLearner)
```

Comment:
Let's not use `.get()` anymore on AlgorithmConfigs. These should never be dicts anymore. We only still do this in our old API stack Policies, where we do operate on dicts (derived from AlgorithmConfig objects).